### DISCUSSION TITLE
How to do streaming of video participants  ?

### DISCUSSION THREAD

**Discussion by PedroSerras50** [webex/webex-js-sdk#3607] [General]
Dear All 
I need to know if it is possible to receive individual video streams from the participants in a meeting. If yes please let me know how to do it
regards
Pedro Serras

**Comment by sreenara** [comment]
@PedroSerras50 we have just released webex version 3.1.0 which contains the "multistream" feature. This feature allows you to receive individual video streams from participants. Please see the guides here:
[Quickstart Guide](https://github.com/webex/webex-js-sdk/wiki/Multistream-Quickstart-Guide)
[Comprehensive Guide](https://github.com/webex/webex-js-sdk/wiki/Multistream-Comprehensive-Guide)

**Comment by PedroSerras50** [comment]
Hello Sreekanth Narayanan,
Thanks a lot for your email
I need to know if you have APIs that could be integarted in Phyton
regards

Pedro Serras
Gerente
________________________________
De: Sreekanth Narayanan ***@***.***>
Enviado: 3 de junho de 2024 17:53
Para: webex/webex-js-sdk ***@***.***>
Cc: Pedro Serras ***@***.***>; Mention ***@***.***>
Assunto: Re: [webex/webex-js-sdk] How to do streaming of video participants ? (Discussion #3607)


@PedroSerras50<https://github.com/PedroSerras50> we have just released webex version 3.1.0 which contains the "multistream" feature. This feature allows you to receive individual video streams from participants. Please see the guides here:
Quickstart Guide<https://github.com/webex/webex-js-sdk/wiki/Multistream-Quickstart-Guide>
Comprehensive Guide<https://github.com/webex/webex-js-sdk/wiki/Multistream-Comprehensive-Guide>

**Reply by sreenara** [reply]
@PedroSerras50 we don't have a python SDK right now for the meeting and calling functions. You would have to directly interact with Webex APIs if you'd like to use python for your application.
